### PR TITLE
add some screen shots for android

### PR DESCRIPTION
**Pull request by payam-zahedi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,11 +20,20 @@ If you can't access Play Store alternatively you can download APK files from Sha
...
-- Copy one of the **Android** config text from the [list of domains](protocols/shadowsocks-v2ray-tls/CONFIGS.md) and open the Shadowsocks app, tab plus button from upper right corner and tab "Import From Clipboard".
+- first Copy one of the **Android** config text from the [list of domains](protocols/shadowsocks-v2ray-tls/CONFIGS.md) and open the Shadowsocks app.
 
 `Example Config:
 ss://YWVzLTI1Ni1nY206WDhDZWN1Y2QyTUZtclJYVGJCcmk@aiohekl.monster:443?plugin=v2ray-plugin%3Bpath%3D%2Fredis%3Bhost%3Daiohekl.monster%3Btls`
+
+- tab plus button from upper right corner and tab.
+<img src="https://user-images.githubusercontent.com/47558577/193402425-7c86138f-fa2b-4087-af95-336215916295.jpg" height="500">
+
+- the tap on the  "Import From Clipboard" item.
```
Comment (farnoodma) [inline README.md:31]: Did you mean "Then tap the .."?

File: README.md [hunk 2]
```diff
@@ -20,11 +20,20 @@ If you can't access Play Store alternatively you can download APK files from Sha
 
  [V2ray Plugin APK](https://github.com/shadowsocks/v2ray-plugin-android/releases/download/v1.3.3/v2ray--universal-1.3.3.apk)
 
-- Copy one of the **Android** config text from the [list of domains](protocols/shadowsocks-v2ray-tls/CONFIGS.md) and open the Shadowsocks app, tab plus button from upper right corner and tab "Import From Clipboard".
+- first Copy one of the **Android** config text from the [list of domains](protocols/shadowsocks-v2ray-tls/CONFIGS.md) and open the Shadowsocks app.
 
 `Example Config:
 ss://YWVzLTI1Ni1nY206WDhDZWN1Y2QyTUZtclJYVGJCcmk@aiohekl.monster:443?plugin=v2ray-plugin%3Bpath%3D%2Fredis%3Bhost%3Daiohekl.monster%3Btls`
+
+- tab plus button from upper right corner and tab.
```
Comment (farnoodma) [inline README.md:28]: "Tap the plus button from the upper right corner" is correct.

File: README.md [hunk 3]
```diff
@@ -20,11 +20,20 @@ If you can't access Play Store alternatively you can download APK files from Sha
...
+<img src="https://user-images.githubusercontent.com/47558577/193402425-7c86138f-fa2b-4087-af95-336215916295.jpg" height="500">
+
+- the tap on the  "Import From Clipboard" item.
+<img src="https://user-images.githubusercontent.com/47558577/193402427-275b568c-d159-4dbe-9fee-5a60570e95d9.jpg" height="500">
+
 - Select the newly added server from the list and tab paper plane button.
```
Comment (farnoodma) [inline README.md:34]: Not one of your changes, but I think it would be great if you change this too: "... and tap the paper plane button."

**Review by farnoodma — CHANGES_REQUESTED** [review]
Thank you for your PR, however, some small things need to be changed before getting merged.

**Comment by payam-zahedi** [comment]
Thanks, @farnoodma.
I updated the PR and fixed issues

**Pull request merged**